go-controller/pkg/ovn/pods.go

```diff
@@ -294,25 +319,28 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 	// transaction.
 	// FIXME: move to the lsp-add transaction once https://bugzilla.redhat.com/show_bug.cgi?id=1806788
 	// is resolved.
-	var uuid string
-	uuid, _, err = util.RunOVNNbctl("get", "logical_switch_port", portName, "_uuid")
-	if err != nil {
-		return fmt.Errorf("error while getting UUID for logical port %s "+
-			"stdout: %q, stderr: %q (%v)", portName, uuid, stderr, err)
+	lsp, err = util.OVNNBDBClient.LSPGet(portName)
+	if err != nil || lsp == nil {
+		return fmt.Errorf("Failed to get the logical switch port: %s from the ovn client, error: %s", portName, err)
 	}
-	if !strings.Contains(uuid, "-") {
-		return fmt.Errorf("invalid logical port %s uuid %q", portName, uuid)
+
+	if !strings.Contains(lsp.UUID, "-") {
+		return fmt.Errorf("invalid logical port %s uuid %q", portName, lsp.UUID)
```

Comment:
stuff like this should probably go away; this is really checking "did `ovn-nbctl` print something unexpected", not "does the database contain invalid data", so it shouldn't be needed any more.

Reply:
I thought this was to check for whether the row got created alright or not and whether the UUID got assigned or not, it seemed a bit un-necessary, and I thought of removing it, but in the end decided to replace the code and keep the logic the same. If we agree, this is pointless, I can take it out.

Reply:
We can kill it. It's a sanity check to make sure the field contains *something* (eg it's basically `uuid != ""`) and if the Go bindings don't do that, then we need to fix the go bindings instead.

Reply:
done.